### ISSUE TITLE
I get stuck on the login screen with a loading button

### ISSUE THREAD

**Issue by jfrick35** [avocode/avocode#230]
Describe your problem here…

-----

```
TypeError: Cannot read property 'ok' of undefined
  at /Users/a6000018/Downloads/Avocode.app/Contents/Resources/app/node_modules/auth/lib/utils/api.js:2:52
  at Request.callback (/Users/a6000018/Downloads/Avocode.app/Contents/Resources/app/node_modules/superagent/lib/node/index.js:746:30)
  at ClientRequest.<anonymous> (/Users/a6000018/Downloads/Avocode.app/Contents/Resources/app/node_modules/superagent/lib/node/index.js:711:10)
  at ClientRequest.emit (events.js:107:17)
  at TLSSocket.socketErrorListener (_http_client.js:271:9)
  at TLSSocket.emit (events.js:129:20)
  at TLSSocket.<anonymous> (_tls_wrap.js:927:18)
  at TLSSocket.emit (events.js:104:17)
  at TLSSocket._finishInit (_tls_wrap.js:457:8)

```

**Comment by petrbrzek** [comment]
Thanks for reporting the issue. We'll look at it. 
Does it occur every time?

**Comment by jfrick35** [comment]
Yep, I'm totally stuck. The explore avocode button works but I can't login
to my account.

**Comment by petrbrzek** [comment]
Do you use some proxy server to get internet access?

**Comment by jfrick35** [comment]
Yes, but usually if I use our guest wifi I can login no problem.

**Comment by jfrick35** [comment]
Also, my photoshop plugin isn't syncing.

**Comment by petrbrzek** [comment]
We're upgrading Electron (internal framework). That should fix it. I'll let you know when we release it.

**Comment by rychlis** [comment]
This should be fixed in Avocode 2.2.0, feel free to re-open the issue if you encounter this again in the latest version.

If you want to use a Proxy server, see our [new article](https://avocode.zendesk.com/hc/en-us/articles/203402352)
